contracts/integrations/test/benchmarks/exchange_benchmarks.ts

```diff
@@ -0,0 +1,98 @@
+import { DummyERC721TokenContract } from '@0x/contracts-erc721';
```

Comment:
Nit - "exchange_benchmarks" is pretty broad. In the interest of keeping these files small, perhaps something more specific / modular?